config/packages/doctrine.yaml

```diff
@@ -14,7 +14,6 @@ doctrine:
         mappings:
             App:
                 is_bundle: false
-                type: attribute
```

Comment:
I'm vaguely surprised this works. Did you happen to find anywhere it was referenced? I remember having to add it, but now I can't really find mention of it anywhere, maybe it gets autodetected now?